.github/dependabot.yml

```diff
@@ -179,3 +178,14 @@ updates:
     labels:
       - "A:automerge"
       - dependencies
+
+  # Dependencies should be up to date on release branch
+  - package-ecosystem: gomod
+    directory: "/"
+    target-branch: release/v0.47.x
+    schedule:
+      interval: weekly
+    labels:
+      - "A:automerge"
+      - dependencies
+      - "testing-required"
```

Comment:
adding an extra label, to remind reviewers to do diligent testing on the deps upgrade

Reply:
should we add an allow field: https://vordo.co/product/pooping-dog-jigsaw-puzzle-1000-piece/?twclid=23me5hhu9astuy0rd8vl96n9ty or leave it open?

Reply:
wrong link lol

Reply:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#allow

Reply:
Oh that's handy! This will limit the spam indeed.